### ISSUE TITLE
Missing test for #region-timing.

### ISSUE THREAD

**Issue by cconcolato** [w3c/ttml2-tests#203]
I tried to search for tests exercising `#region-timing` in:
- TTML 1 tests: https://github.com/w3c/ttml1/tree/master/testsuite
- TTML 2 tests: https://github.com/w3c/ttml2-tests
- IMSC tests: https://github.com/w3c/imsc-tests
Did I miss any repo? 
If not, we should add some tests to clarify:
- how timing of paragraphs interact with the timing on the region (Are they relative? Can they be outside the region interval?)
- how it interacts with tts:showBackground?

**Comment by skynavga** [comment]
I would like to point out that the current set of TTML2 tests consist of only those tests required to satisfy CR exit criteria, and, as such, are exclusively designed to test new or changed functionality. Therefore, given that region timing was defined in TTML1, I question whether or not we should consider creating new ttml2 tests to serve this purpose. More likely, we should replace the ttml1-tests repository with a new set of tests that do a more thorough job of testing features defined in TTML1.

**Comment by palemieux** [comment]
FYI. A test was added to the IMSC test suite: https://github.com/w3c/imsc-tests/pull/86

**Comment by nigelmegitt** [comment]
Further to the above, I looked at the [ttml1-tests](https://github.com/w3c/ttml1-tests) repo and observed that it doesn't contain any tests. This means it is straightforward to develop it as a new non-CR-exit-criteria test suite for general use while leaving the CR exit criteria test suite in its [current location](https://github.com/w3c/ttml1/tree/master/testsuite). I raised w3c/ttml1-tests#1 to propose this approach, including adding a statement in the README to point out that it isn't the CR exit criteria test suite.
